src/NzbDrone.Core/Parser/IsoLanguages.cs

```diff
@@ -27,6 +27,7 @@ public static class IsoLanguages
                                                                new IsoLanguage("fi", "fin", Language.Finnish),
                                                                new IsoLanguage("tr", "tur", Language.Turkish),
                                                                new IsoLanguage("pt", "por", Language.Portuguese),
+                                                               new IsoLanguage("br", "bra", Language.Brazilian_Portuguese),
```

Comment:
This list is ISO 639-2 language codes, not country codes. 'bra' is the language code of Braj Bhāshā, a Western Hindi language.

Reply:
From what I've searched, there isn't a ISO 639-2 for Brazilian Portuguese:
![image](https://user-images.githubusercontent.com/49880491/169046680-3431f7a8-5436-4c48-8298-732a24ad75f0.png)

So we just use the same code as "Portuguese"?

Reply:
And also, in this snippet of the code, is specifying that the ISO 639-1 code will only have two digits, 
```
if (isoCode.Length == 2)
            {
                //Lookup ISO639-1 code
                return All.SingleOrDefault(l => l.TwoLetterCode == isoCode);
            }
```

But from what I've searched, Brazilian Portuguese ISO 639-1 code is "pt-BR", as "br" is the code for Breton. 

So what is the approach here?

Reply:
Radarr had to make various other changes as noted in that commit to support dual language codes

Reply:
> Radarr had to make various other changes as noted in that commit to support dual language codes

How bad would it be if we just used the same ISO code for both Portuguese and Brazilian Portuguese?

It wasn't my intention to change the entire parse ISO logic of the project.

Reply:
> pt is a [language code](https://en.wikipedia.org/wiki/Language_code) for Portuguese, defined by [ISO standards](https://en.wikipedia.org/wiki/ISO_standard) (see [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) and [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)). There is no ISO code for spoken or written Brazilian Portuguese.

> pt-BR is a [language code](https://en.wikipedia.org/wiki/Language_code) for the Brazilian Portuguese, defined by [Internet standards](https://en.wikipedia.org/wiki/Internet_standard) (see [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag)).

Source: https://en.wikipedia.org/wiki/Brazilian_Portuguese

Looks like there aren't separate language codes for Brazilian Portuguese.

> How bad would it be if we just used the same ISO 639-3 code for both Portuguese and Brazilian Portuguese? We can use "br" and "por".

You can't just use whatever you want there, `br` is used for Breton, even if it's not in Sonarr you can't just usurp it to make it work.

To do this right, you're going to need to do what Radarr did, there isn't a shortcut here that will work.